### PR TITLE
Content page design

### PR DESCRIPTION
**Pull request by Cajan19**
Made some basic layout (Appbar, footer, content) /w placeholders.

### CODE REVIEW

File: frontend-nannyway/src/components/pages/contentPages/ChildInCareOverview.js [hunk 1]
```diff
@@ -1,9 +1,79 @@
...
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        flexGrow: 1,
+    },
+    main: {
+        display: 'flex',
+        flexDirection: 'column',
+        minHeight: '70vh',
+    },
```
Comment (marwinburesch) [inline frontend-nannyway/src/components/pages/contentPages/ChildInCareOverview.js:18]: I don't see this class used anywhere

File: frontend-nannyway/src/components/pages/contentPages/ChildInCareOverview.js [hunk 2]
```diff
@@ -1,9 +1,79 @@
...
+    nannywaytypo: {
+        fontFamily: "Finger Paint",
+    },
+    basictypo: {
+        fontFamily: "Open Sans",
+    },
```
Comment (marwinburesch) [inline frontend-nannyway/src/components/pages/contentPages/ChildInCareOverview.js:24]: same for these, I don't see them used anywhere on this page, did you add these for future components?

File: frontend-nannyway/src/components/pages/contentPages/ChildInCareOverview.js (hunk 1 above)
Comment (Cajan19) [reply frontend-nannyway/src/components/pages/contentPages/ChildInCareOverview.js]: Whoops.. deleted now.

File: frontend-nannyway/src/components/pages/contentPages/ChildInCareOverview.js (hunk 2 above)
Comment (Cajan19) [reply frontend-nannyway/src/components/pages/contentPages/ChildInCareOverview.js:19]: Gave the classes some use ;)

**Pull request merged**